Dropdown - ViewDestroyedError

**Minimal reproduction of the problem with instructions**
Using `@angular/forms`'s `FormBuilder` approach, set a `p-dropdown` to disabled

```
error-handler.ts:106 Error: ViewDestroyedError: Attempt to use a destroyed view: detectChanges
    at viewDestroyedError (core.js:25516)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:39361)
    at checkAndUpdateView (core.js:38376)
    at callWithDebugContext (core.js:39716)
    at Object.debugCheckAndUpdateView [as checkAndUpdateView] (core.js:39299)
    at ViewRef_.detectChanges (core.js:27092)
    at Dropdown.set [as disabled] (dropdown.js:116)
    at Dropdown.push../node_modules/primeng/components/dropdown/dropdown.js.Dropdown.setDisabledState (dropdown.js:256)
    at forms.js:2612
    at forms.js:3361
```

* **Angular version:** 8.1.2
* **PrimeNG version:** 8.0.1

--- a/src/app/components/dropdown/dropdown.ts
+++ b/src/app/components/dropdown/dropdown.ts
@@ -1,6 +1,6 @@
 import {ScrollingModule, CdkVirtualScrollViewport} from '@angular/cdk/scrolling';
 import {NgModule,Component,ElementRef,OnInit,AfterViewInit,AfterContentInit,AfterViewChecked,OnDestroy,Input,Output,Renderer2,EventEmitter,ContentChildren,
-        QueryList,ViewChild,TemplateRef,forwardRef,ChangeDetectorRef,NgZone} from '@angular/core';
+        QueryList,ViewChild,TemplateRef,forwardRef,ChangeDetectorRef,NgZone,ViewRef} from '@angular/core';
 import {trigger,state,style,transition,animate,AnimationEvent} from '@angular/animations';
 import {CommonModule} from '@angular/common';
 import {SelectItem} from '../common/selectitem';
@@ -264,7 +264,9 @@ export class Dropdown implements OnInit,AfterViewInit,AfterContentInit,AfterView
             this.focused = false;
         
         this._disabled = _disabled;
-        this.cd.detectChanges();
+        if(!(this.cd as ViewRef).destroyed) {
+            this.cd.detectChanges();
+        }
     }
 
     overlay: HTMLDivElement;

